### PR TITLE
refactor(packages) - remove "packages" endpoint, use "archive" instead

### PR DESCRIPTION
**Pull request by amagdas**

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1: 
There are 2 behave tests that should be updated/fixed

File: server/apps/archive/archive.py [hunk 1]
```diff
@@ -128,6 +130,8 @@ def update_word_count(doc):
 
 
 class ArchiveService(BaseService):
+    packageService = PackageService()
```
Comment (sivakuna-aap) [inline server/apps/archive/archive.py]: It might be better to move this to `__init__()`

Comment (ioanpocol) [inline server/apps/archive/archive.py]: The PackageService is not public, it is used only here.

**Comment by mugurrus** [comment]
:+1:

**Comment by ioanpocol** [comment]
Disabled e2e test related to package and created a new PR:
https://github.com/superdesk/superdesk/pull/137

The package related e2e will be fixed by:
https://github.com/superdesk/superdesk/pull/101

**Pull request closed without merging**